uk.co.compucorp.civicrm.tasksassignments/js/src/crm-tasks-workflows/decorators/route.decorator.js

```diff
@@ -34,6 +43,24 @@
       return activityOptionsResolver;
     }
 
+    /**
+     * Returns a resolver for the list of workflow case types.
+     *
+     * @return {Function} the function that will resolve the workflow case types.
+     */
+    function getResolverForWorkflowTypes () {
+      workflowTypesResolver.$inject = ['crmApi'];
+
+      function workflowTypesResolver (crmApi) {
```

Comment:
```js
var workflowTypesResolver = function (crmApi)
  ...
}
```
That would be probably easier to read, you define a var, then return it. It will also ensure the order of code execution.

Reply:
I have refactored it this way to avoid injection issues:

```js
function getResolverForWorkflowTypes () {
  return ['crmApi', function workflowTypesResolver (crmApi) {
    // ...
  });
}
```